src/Controller/ProfilerController.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Overblog\GraphQLBundle\Controller;
+
+use GraphQL\Utils\SchemaPrinter;
+use Overblog\GraphQLBundle\Request\Executor as RequestExecutor;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+use Symfony\Component\HttpKernel\Profiler\Profiler;
+use Symfony\Component\Routing\RouterInterface;
+use Twig\Environment;
+
+class ProfilerController
+{
+    private $profiler;
+    private $twig;
+    private $endpointUrl;
+    private $requestExecutor;
+    private $queryMatch;
+
+    public function __construct(Profiler $profiler = null, Environment $twig = null, RouterInterface $router, RequestExecutor $requestExecutor, string $queryMatch = null)
```

Comment:
Optional parameters cannot stand before required:

```suggestion
    public function __construct(?Profiler $profiler, ?Environment $twig, RouterInterface $router, RequestExecutor $requestExecutor, string $queryMatch = null)
```